proto/README.md

```diff
@@ -79,3 +84,22 @@ Field | Unit | Type | Description
 --- | --- | --- | ---
 positions | meters | Vector3 (list) | List of object’s predicted XYZ position.
 reachable_set | meters | [ReachableSet](https://github.com/seoulrobotics/sensr_proto/blob/master/type.proto) | List of object's predicted position including uncertainty sorted by elapsed time.
+
+### ReplayInfo
+
+Field | Unit | Type | Description
+--- | --- | --- | ---
+current_index | - | int32 | Current frame index of the playing replay file.
+
+### RecordingInfo
+
+Field | Unit | Type | Description
+--- | --- | --- | ---
+saving_progress | - | float | Progress of saving recorded file into disk.
```

Comment:
Do we need to explain the range (0.0 to 1.0)?

Reply:
Better than nothing.
Will add it.